app/views/posts/index.html.erb

```diff
@@ -1,2 +1,27 @@
-<h2>I come in peace</h2>
-<p>Lets go to the Masjid</p>
\ No newline at end of file
+<h2>The User Post</h2>
+<div class="container">
+        <div class="image">
+          <img src="<%= @user.Photo %>" class="img-user" alt="<%= @user.Name %>" />
+        </div>
+        <div class="details">
+          <h1 class="details-head"><%= @user.Name %></h1>
+          <p class="details-para">Number of posts: <%= @user.PostsCounter || 0 %></p>
+        </div>
+</div>
+
+<% @user.posts.each do |post| %>
```

Comment:
- [ ] Let's render the 5 recent comments for each post as the requirements stipulated.

- [ ] I also discovered that I'm unable to create comments for a post. Kindly check the screenshot below and resolve that error.



![Screenshot from 2022-10-07 17-12-39](https://user-images.githubusercontent.com/63087491/194600362-cfd0a424-e6a5-4d2f-8943-5374631cd4f3.png)

Reply:
@FrederickMih I gave the post index view to have 5 most recent comments to go in line with the project requirement

Reply:
@FrederickMih I rectified the comment-adding aspect and you can now add a comment
![comment](https://user-images.githubusercontent.com/78634841/194652816-824b596b-90cd-4727-9172-842e0acb94e1.png)